ocaml/typing/shape.mli

```diff
@@ -82,12 +82,12 @@ val print : Format.formatter -> t -> unit
 val for_unnamed_functor_param : var
 val fresh_var : ?name:string -> Uid.t -> var * t
 
-val var : Uid.t -> Ident.t -> t
```

Comment:
This change doesn't seem necessary and makes the interface worse, making it much more likely to forget to include a uid.